utils/httpsAgent.ts

```diff
@@ -0,0 +1,23 @@
+import * as https from "https";
+import { getKeepAliveAgentOptions } from "@pagopa/ts-commons/lib/agent";
+import nodeFetch from "node-fetch";
+
+export const getFetchWithClientCertificate: (
+  env: NodeJS.ProcessEnv,
+  clientCertPrefix: string
+) => typeof fetch = (env, clientCertPrefix) => {
+  const httpsAgent = new https.Agent({
+    ...getKeepAliveAgentOptions(env),
+    cert: env[`${clientCertPrefix}_CLIENT_CERT`],
+    key: env[`${clientCertPrefix}_CLIENT_KEY`]
```

Comment:
I think we can just use `getFetch(env, options)` and set certificates into `options`. Can we?

Reply:
Certificate options like "cert" are not available in extra options, due to old node.js version dependency in ts-commons project. We could re-use only the env-configuration function.